tools/oesign/oesign.c

```diff
@@ -346,43 +344,127 @@ int oesign(
      * found or fails the load.
      */
     OE_CHECK_ERR(
-        oe_read_oeinfo_sgx(enclave, &props),
+        oe_read_oeinfo_sgx(enclave, props),
         "Failed to load enclave: %s: result=%s (%#x)",
         enclave,
         oe_result_str(result),
         result);
 
     /* Merge the loaded configuration file with existing enclave properties */
-    _merge_config_file_options(&props, &options);
+    _merge_config_file_options(props, &options);
 
     /* Check whether enclave properties are valid */
     {
         const char* field_name;
         OE_CHECK_ERR(
-            oe_sgx_validate_enclave_properties(&props, &field_name),
+            oe_sgx_validate_enclave_properties(props, &field_name),
             "Invalid enclave property value: %s",
             field_name);
     }
 
+    result = OE_OK;
+
+done:
+    return result;
+}
+
+oe_result_t _get_sgx_enclave_hash(
+    const char* enclave,
+    oe_sgx_enclave_properties_t* props,
```

Comment:
Make const.  Also suggest not abbreviating "properties".

```suggestion
    const oe_sgx_enclave_properties_t* properties,
```

Reply:
Made `const` and replaced all existing instances of `props` with `properties` that this instance was originally copied from